kerl

```diff
@@ -1291,50 +1251,67 @@ kerl_deactivate() {
     fi
     unset KERL_ENABLE_PROMPT
     unset KERL_PROMPT_FORMAT
+    set +o allexport
 }
 kerl_deactivate nondestructive
 
+_KERL=$(command -v kerl)
+
+if [ -n "\$_KERL" ]; then
+  _KERL_VERSION=\$(\$_KERL version)
+fi
+
+if [ -n "\$_KERL_VERSION" ] && [ "\$_KERL_VERSION" != "$KERL_VERSION" ]; then
+  echo "WARNING: this Erlang/OTP installation appears to be stale. Please consider reinstalling."
+  echo "         It was created with kerl $KERL_VERSION, and the current version is \$_KERL_VERSION."
+fi
+
+set -o allexport
 _KERL_SAVED_REBAR_PLT_DIR="\$REBAR_PLT_DIR"
-export _KERL_SAVED_REBAR_PLT_DIR
-_KERL_PATH_REMOVABLE="$absdir/bin"
+_KERL_PATH_REMOVABLE="$directory/bin"
 PATH="\${_KERL_PATH_REMOVABLE}:\$PATH"
-export PATH _KERL_PATH_REMOVABLE
-_KERL_MANPATH_REMOVABLE="$absdir/lib/erlang/man:$absdir/man"
+_KERL_MANPATH_REMOVABLE="$directory/lib/erlang/man:$directory/man"
 MANPATH="\${_KERL_MANPATH_REMOVABLE}:\$MANPATH"
-export MANPATH _KERL_MANPATH_REMOVABLE
-REBAR_PLT_DIR="$absdir"
-export REBAR_PLT_DIR
-_KERL_ACTIVE_DIR="$absdir"
-export _KERL_ACTIVE_DIR
-_KERL_ERL_CALL_REMOVABLE=\$(\\find $absdir -type d -path "*erl_interface*/bin")
+REBAR_PLT_DIR="$directory"
+_KERL_ACTIVE_DIR="$directory"
+_KERL_ERL_CALL_REMOVABLE=\$(\\find $directory -type d -path "*erl_interface*/bin")
 PATH="\${_KERL_ERL_CALL_REMOVABLE}:\$PATH"
-export PATH _KERL_ERL_CALL_REMOVABLE
 # https://twitter.com/mononcqc/status/877544929496629248
-export _KERL_SAVED_ERL_AFLAGS=" \$ERL_AFLAGS"
+_KERL_SAVED_ERL_AFLAGS=" \$ERL_AFLAGS"
 kernel_history=\$(echo "\$ERL_AFLAGS" | \\grep 'kernel shell_history' || true)
 if [ -z "\$kernel_history" ]; then
-    export ERL_AFLAGS="-kernel shell_history enabled \$ERL_AFLAGS"
+    ERL_AFLAGS="-kernel shell_history enabled \$ERL_AFLAGS"
 fi
 # shellcheck source=/dev/null
 if [ -f "$KERL_CONFIG" ]; then . "$KERL_CONFIG"; fi
 if [ -n "\$KERL_ENABLE_PROMPT" ]; then
     _KERL_SAVED_PS1="\$PS1"
-    export _KERL_SAVED_PS1
     if [ -n "\$KERL_PROMPT_FORMAT" ]; then
-        FRMT="\$KERL_PROMPT_FORMAT"
```

Comment:
Renamed in order to be "safely" exported (though this is a local-only variable).